app/views/meditations/_course.html.slim

```diff
@@ -0,0 +1,24 @@
+
+- translations = 'meditations.course'
+- gradient_classes = 'gradient--left gradient--blue'
+
+.course id="course"
```

Comment:
Could you prefix these class names with `realisation_` to avoid any name collisions in the future. So do `realisation__course__content` instead of `course__content`